README.md

```diff
@@ -135,6 +135,52 @@ The cluster creation and retrieving connection info can be done directly in comm
   hzc --cluster-name <CLUSTER NAME> --cloud-token <DISCOVERY TOKEN>
   ```
 
+## SSL Configuration
+
+You can use the following configuration file to enable SSL support:
+```
+ssl:
+    servername: "HOSTNAME-FOR-SERVER"
+    # or: insecureskipverify: true
+hazelcast:
+  cluster:
+    security:
+      credentials:
+        username: "OPTIONAL USERNAME"
+        password: "OPTIONAL PASSWORD"
+    name: "CLUSTER-NAME"
+    network:
+      addresses:
+        - "localhost:5701"
+```
+
+Mutual authentication is also supported:
+```
+ssl:
+    servername: "HOSTNAME-FOR-SERVER"
+    # insecureskipverify: true
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast: #
+```
+
+Cloud SSL configuration:
+```
+ssl:
+    servername: "hazelcast.cloud"
+    capath: "/tmp/ca.pem"
+    certpath: "/tmp/cert.pem"
+    keypath: "/tmp/key.pem"
+    keypassword: "PASSWORD FOR THE KEY"
+hazelcast: #
```

Comment:
we can delete this
```suggestion
hazelcast:
```

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/43/commits/7e85138d4528191eb23ce12620e392464ac1a663